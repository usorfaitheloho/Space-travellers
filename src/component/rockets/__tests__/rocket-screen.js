import React from 'react';
import { Provider } from 'react-redux';
import render from 'react-test-renderer';
import store from '../../../redux/configureStore';
import Rocket from '../rocket';

test('Rocket screen load correctly', () => {
  const tree = render.create(
    <Provider store={store}>
      <Rocket />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
