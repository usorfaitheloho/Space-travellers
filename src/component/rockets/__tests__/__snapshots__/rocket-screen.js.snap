// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket screen load correctly 1`] = `
<div
  className="rocket-container"
/>
`;
